deploy/terraform-custom-datacommons/modules/terraform.tfvars.sample

```diff
@@ -22,3 +22,6 @@ project_id  = "your-gcp-project"
 
 # User provided namespace. Example: "dev", "prod", "dan-dev"
```

Comment:
Explain what this will be used for.

Reply:
I mistakenly merged this PR before addressing this comment. See the follow-up https://github.com/datacommonsorg/website/pull/4565